requirements/AUR_RT-0020/Taste-Leon3-Runtime-target.txt

```diff
@@ -0,0 +1,2 @@
+Requirement fulfilled, document reference:
```

Comment:
I would expect a bit of explanation/comment, for better readability. This applies also to other design requirements,. For example:

TASTE Leon3 Runtime targets Leon3 processor, as per ($document reference).
TASTE Leon3 Runtime targets ESA's RTEMS SMP QDP, as per ($document reference). The exact version is RTEMS 6 SPARC GR712RC SMP 4 (see installation script in TASTE repository) (it would be great if we could point to a document, like SDP or SRF, but we don't have one :/ )
....
TASTE Leon3 Runtime uses X for implementing timers, as per $document reference (SDD). X provides the resolution of Y, as per $document reference 2 (GR712....). It would be ideal to also provide a reference to the exact configuration, but we don;'t have the appropriate document, so we will have to do without it.

Reply:
Done